### DISCUSSION TITLE
Not able to sync

### DISCUSSION THREAD

**Discussion by glukkon24** [Chia-Network/chia-blockchain#8077] [Support]
I have been running Chia for several months. I had to reboot my computer and Chia would not come back on so I did the old delete .Chia folder and started up, shutdown and then put my old database in. It will not start with the old database so I got one from someone else and it still will not start. I can sync from the beginning but whenever I try to insert a complete or close to complete database it just has the spinning. I can take out the complete database and put back in the new one and it runs fine. It will take me a couple days to sync back up. Does anyone have a solution or detailed instructions on what I am doing wrong and what I should be doing?

**Comment by BrandtH22** [comment]
@glukkon24 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
